go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/offchainlabs/nitro
 
-go 1.20
+go 1.21
```

Comment:
Upstream is using `1.20` so I think we should stick with that unless we have a very compelling reason not to.

https://github.com/OffchainLabs/nitro/blob/master/go.mod#L3